conf/messages.en

```diff
@@ -64,12 +64,16 @@ viewArrivalNotifications.table.trader.name = Trader name
 viewArrivalNotifications.table.office      = Office
 viewArrivalNotifications.table.procedure   = Procedure
 viewArrivalNotifications.table.status      = Status
+unloadingPermission.link.title = Make unloading remarks
```

Comment:
Can we move this outside of the table keys

Reply:
Maybe rename it to something like _movement.action.unloadingPermission_ and put it with the other movement. keys

Reply:
changed to viewArrivalNotifications.table.action.unloadingRemarks in line with other movement keys

Reply:
Done